projeto-pokedex/src/pages/Pokedex.js

```diff
@@ -2,27 +2,28 @@ import React, { useContext, useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { GlobalContext } from "../global/GlobalContext";
 import {goToDetailPokemon, goToBack} from "../routes/coordinator"
-import { CardPokemon, ContainerPokemon, CardButton, CardIdName, Type } from "../components/StyledListPokemons";
+import { CardPokemon, ContainerPokemon, CardButton, CardIdName, Type, ButtonCapturar } from "../components/StyledListPokemons";
 
 function Poquedex() {
 
-  // const {pokedex, setPokedex } = useContext(GlobalContext)
-  // console.log(pokedex)
-
-  const navigate = useNavigate()
-
-
-  const {detailedListPokemon, setDetailedListPokemon} = useContext(GlobalContext);
-  let capturedPokemonsLocalStorage = localStorage.getItem("capturedPokemons") ? JSON.parse(localStorage.getItem("capturedPokemons")) : {}
+  const {pokedex, setPokedex } = useContext(GlobalContext)
+  const {capturedPokemons, setCapturedPokemons} = useContext(GlobalContext)
+  console.log(pokedex)
 
+  
 
-  const [capturedPokemons, setCapturedPokemons] = useState(capturedPokemonsLocalStorage)
+  const removeFromPokedex = (pokemonId) => {
+    let newCapturedPokemons = JSON.parse(JSON.stringify(capturedPokemons))
+    let pokedexCopy = pokedex
```

Comment:
This line doesn't do anything since objects are pass-by-reference (rather than pass-by-value) in Javascript (https://codeburst.io/explaining-value-vs-reference-in-javascript-647a975e12a0).
Might as well do `delete pokedex[pokemonId]`. Keep in mind tho that this won't trigger a React rerender of the (currently visible) components.